server/database/modules/donor.ts

```diff
@@ -0,0 +1,42 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../config/connection';
+
+const donor = sequelize.define('donors', {
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  password: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  address: {
+    type: DataTypes.STRING,
+  },
```

Comment:
You need to add that it is not allowed to be null, right?
The same for the family model.